android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/components/DropdownAnswer.kt

```diff
@@ -0,0 +1,98 @@
+package co.nimblehq.ic.kmm.suv.android.ui.components
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.Divider
+import androidx.compose.material.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.graphicsLayer
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.times
+import androidx.compose.ui.util.lerp
+import co.nimblehq.ic.kmm.suv.android.ui.theme.Typography
+import com.google.accompanist.pager.ExperimentalPagerApi
+import com.google.accompanist.pager.VerticalPager
+import com.google.accompanist.pager.calculateCurrentOffsetForPage
+import com.google.accompanist.pager.rememberPagerState
+import kotlin.math.absoluteValue
+
+@OptIn(ExperimentalPagerApi::class)
+@Composable
+fun DropdownAnswer(
+    choices: List<String>,
+    onIndexChange: (Int) -> Unit
+) {
+    val padding = 56.dp
+    val pagerState = rememberPagerState()
+    LaunchedEffect(pagerState.currentPage) {
+        onIndexChange(pagerState.currentPage)
+    }
+    Box {
+        VerticalPager(
+            count = choices.size,
+            state = pagerState,
+            contentPadding = PaddingValues(vertical = padding),
+            modifier = Modifier
+                .height(170.dp)
+                .fillMaxWidth()
+        ) { page ->
+            val choice = choices[page]
+            val isHighlight = pagerState.currentPage == page
+            Box(
+                contentAlignment = Alignment.Center,
+                modifier = Modifier
+                    .height(56.dp)
```

Comment:
please use common dimension

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/121/commits/65286cf9f85f749815db7767583f78ed841ced98